app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotWearingMaskError
+from app.errors import NotVaccinatedError, OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" in visitor:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
`if “vaccine” not in visitor:`
`if visitor["vaccine"]["expiration_date"] < date.today(): `